variables.tf

```diff
@@ -232,7 +246,6 @@ variable "log_analytics_solution_id" {
   type        = string
   description = "(Optional) Existing azurerm_log_analytics_solution ID. Providing ID disables creation of azurerm_log_analytics_solution."
   default     = null
-  nullable    = true
```

Comment:
I understand you removed this line because of a `tflint` error. But we should fix the CI in this case. Can you revert these 3 changes where you remove the lines `nullable = true` from the variables that are not in scope with this PR ? thanks